app/views/thredded/shared/_page.html.erb

```diff
@@ -1,6 +1,7 @@
 <div id="thredded--container" class="thredded--main-container <%= yield(:thredded_page_id) %><%= ' thredded--is-moderator' if moderatable_messageboards_ids.present? %>" data-thredded-page-id="<%= yield(:thredded_page_id) %>">
   <%= render 'thredded/shared/header' %>
   <%= render 'thredded/shared/flash_messages' %>
+  <div id="thredded--definitions" data-thredded-root="<%= thredded.root_path%>"></div>
```

Comment:
> and may as well make it a url, rather than a path

A path is a (domain-relative) URL. Since paths are friendlier to domain-mirroring, it's better to keep it a path.

>Do you think maybe worth making thredded--container a helper as it's rather hard to read in erb? 

👍